src/main/java/com/faforever/api/data/domain/Player.java

```diff
@@ -22,14 +25,15 @@
 @SharePermission(expression = "Prefab.Role.All")
 @Setter
 @Type(Player.TYPE_NAME)
-public class Player extends Login {
+public class Player extends Login implements OwnableEntity {
 
   public static final String TYPE_NAME = "player";
   private Ladder1v1Rating ladder1v1Rating;
   private GlobalRating globalRating;
   private List<ClanMembership> clanMemberships;
   private List<NameRecord> names;
   private List<AvatarAssignment> avatarAssignments;
+  private List<Vote> votes;
```

Comment:
Remove this list. It's not wrong, but Hibernate will load it everytime we query a player and elide has to check for every entry if the requester is allowed it to be visible. The player is allowed to see his votes only, so he can query them from the Vote entity itself.